build.cake

```diff
@@ -226,7 +226,7 @@ Task("Build")
         MSBuild("./Xamarin.Forms.Platform.UAP/Xamarin.Forms.Platform.UAP.csproj",
                     GetMSBuildSettings()
                         .WithRestore()
-                        .WithProperty("DisableEmbeddedXbf", "true"));
+                        .WithProperty("DisableEmbeddedXbf", "false"));
```

Comment:
is this supposed to be changed?

Reply:
Yea

This is only used for cases when you're creating the nugets from the command line not the CI

Basically it's telling it to "Not Not Embed Xbf files" which is what we want opposed to just "Not Embedding them" :-)